src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.discussions;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.url.UrlBuilder'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)